src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -219,11 +219,14 @@ namespace HighTempRadiantSystem {
 
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		static bool GetInputFlag( true ); // First time, input is "gotten"
```

Comment:
@mjwitte Yeah, probably need an example as I am simply following examples from others at this point and I don't always catch the finer points.  Thanks in advance.